doc/COMPILING/COMPILING.md

```diff
@@ -597,40 +597,25 @@ Clang by default uses MSVC on Windows, but also supports the MinGW64 library. Si
 
 There are reports of CDDA building fine on recent OpenBSD and FreeBSD machines (with appropriately recent compilers), and there is some work being done on making the `Makefile` "just work", however we're far from that and BSDs support is mostly based on user contributions. Your mileage may vary. So far essentially all testing has been on amd64, but there is no (known) reason that other architectures shouldn't work, in principle.
 
-### Building on FreeBSD/amd64 10.1 with the system compiler
+### Building on FreeBSD/amd64 13.0 with the system compiler
 
-FreeBSD uses clang as the default compiler as of 10.0, and combines it with libc++ to provide C++14 support out of the box. You will however need gmake (examples for binary packages):
+FreeBSD uses clang as the default compiler as of 10.0, and combines it with libc++ to provide C++14 support out of the box.
 
-`pkg install gmake`
+Install the following with pkg (or from Ports):
+
+`pkg install gmake libiconv`
 
 Tiles builds will also require SDL2:
 
-`pkg install sdl2 sdl2_image sdl2_mixer sdl2_ttf`
+`pkg install sdl20 sdl2_image sdl2_mixer sdl2_ttf`
 
-Then you should be able to build with something like this (you can of course set CXXFLAGS and LDFLAGS in your .profile or something):
+Then you should be able to build with something like this:
 
 ```
-export CXXFLAGS="-I/usr/local/include" LDFLAGS="-L/usr/local/lib"
 gmake # ncurses builds
 gmake TILES=1 # tiles builds
```

Comment:
Does RELEASE=1 work/make sense? Generally that's required for a modestly well performing build.

Reply:
Don't see why it wouldn't, but I can confirm. Also, I've never seen the tests build successfully in FreeBSD 13 or 12. Considered that a separate issue to look into, so I didn't include it here. Otherwise everything is as expected, and the game runs fine.

Reply:
The tests don't _build_ successfully, but the main executable does? Interesting.

Reply:
Correct. Running `gmake -C tests` directly is required to start building the tests, but there are still some issues when compiling.